lab_3_ann_retriever/main.py

```diff
@@ -49,6 +50,15 @@ def calculate_distance(query_vector: Vector, document_vector: Vector) -> float |
 
     In case of corrupt input arguments, None is returned.
     """
+    if query_vector is None or document_vector is None:
+        return None
+    if not query_vector or not document_vector:
+        return 0.0
```

Comment:
![2024-11-12_19-36-21](https://github.com/user-attachments/assets/d826e344-2f68-4d1f-8234-b602a12d011f)
Because if I return None, there is an error that says that 0.0 was expected. I don't know, why 0.0. With None I have an error in CalculateDistance.test_calculate_distance_for_empty_vectors

Reply:
ok